invenio_mail/tasks.py

```diff
@@ -27,7 +30,52 @@ def send_email(data):
        `custom serializer <http://docs.celeryproject.org/en/latest/
        userguide/calling.html#serializers>`__
        can be created if attachments are really needed.
+
+       This version adds attachments by putting them into a base64 encoded string.
+       This is not an optimal solution, it might get problematic if they are too
+       large to handle by the messaging queue, so check for a maximum size beforehand.
     """
     msg = Message()
-    msg.__dict__.update(data)
+    msg.subject = data.get("subject")
```

Comment:
Why explicitly setting all (or some) of the fields? Is it because of the manipulation of the `attachments`?

Reply:
It is because I extended the dict with the attachments in a form that is not compatible with flask-mail. Another possibility would have been to remove the attachments from the dict and leave the update of tha data as it was.

Reply:
I have updated the code in the PR.